src/style.css

```diff
@@ -0,0 +1,117 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  color: #060b35;
+}
+
+h1,
+h2,
+body {
+  margin: 0;
```

Comment:
Goole `how selector * works` (use MDN) to find out why you can add `margin: 0` to that selector and delete all `margin: 0` below and above =)

Reply:
I heard that I should not add nothing to this selector but box-sizing property

Reply:
Box sizing and reset margin to 0 is ok!
this selector is very strong and resource-intensive, so you can use only such light styles with it.
Avoid to use `display: block` || `font-family: "Robot"` || `align-items` || `hight`.
But it's really ok to use it for the reset! =)

Reply:
ok, i got it, gonna change my code right now))